docs/en/modules/wifi.md

```diff
@@ -270,6 +270,24 @@ wifi.sta.changeap(4)
 - [`wifi.sta.getapinfo()`](#wifistagetapinfo)
 - [`wifi.sta.getapindex()`](#wifistagetapindex)
 
+## wifi.sta.clearconfig()
+
+Clears the currently saved WiFi station configuration. This will erase the saved configuration from the flash (useful for 
+factory-reset scenarios, or to prepare for [End-User Setup](enduser-setup) so that the SoftAP can lock onto a single channel)
```

Comment:
>useful for factory-reset scenarios

The function `node.restore()` should be used to ensure complete erasure of WiFi configuration, since it completely overwrites the system parameter sectors that store WiFi configuration and other configuration information.